### ISSUE TITLE
The addon needs an icon and screenshots

### ISSUE THREAD

**Issue by Razzeee** [xbmc/notrobro#14]
We should also think if we want the author to be "Team Kodi"

**Comment by LordGameleo** [comment]
Should we also add fanart?

**Comment by Razzeee** [comment]
Fanart would be good, but really hard to design that for a tool like this.

**Comment by LordGameleo** [comment]
So what do you suggest?........ We can add some icon like using initial 
example:
![1](https://user-images.githubusercontent.com/40512667/54074623-c7814980-42ba-11e9-9ebc-24dd9def6f57.png)

**Comment by LordGameleo** [comment]
@Razzeee We have removed fanart and icon from addon in PR #19 
So I think this issue is not required as such.

**Comment by Razzeee** [comment]
Yes, not required, but should be created to have the best possible gui for users.

**Comment by LordGameleo** [comment]
we can create one default icon and fanart with letter K (for Kodi).... What do you think about it.

**Comment by Razzeee** [comment]
No, let's just wait for someone with a decent logo idea :)
